beacon-chain/rpc/beaconv1/config.go

```diff
@@ -11,8 +12,37 @@ import (
 )
 
 // GetForkSchedule retrieve all scheduled upcoming forks this node is aware of.
-func (bs *Server) GetForkSchedule(ctx context.Context, req *ptypes.Empty) (*ethpb.ForkScheduleResponse, error) {
-	return nil, errors.New("unimplemented")
+func (bs *Server) GetForkSchedule(ctx context.Context, _ *ptypes.Empty) (*ethpb.ForkScheduleResponse, error) {
+	ctx, span := trace.StartSpan(ctx, "beaconv1.GetForkSchedule")
+	defer span.End()
+
+	schedule := params.BeaconConfig().ForkVersionSchedule
+	if len(schedule) == 0 {
+		return &ethpb.ForkScheduleResponse{
+			Data: make([]*ethpb.Fork, 0),
+		}, nil
+	}
+
+	epochs := sortedEpochs(schedule)
+	forks := make([]*ethpb.Fork, len(schedule))
+	var previous, current []byte
+	for i, e := range epochs {
```

Comment:
This assumes sequentiality. What about?

![Screen Shot 2021-01-27 at 7 44 01 AM](https://user-images.githubusercontent.com/21316537/106015660-8becf580-6073-11eb-94b6-22fc21d4b21f.png)

I don't think `ForkVersionSchedule` in config is spec'ed correctly. It should be contain both current and previous versions

Reply:
The forks in the config are scheduled forks. Isn't it reasonable to assume sequentiality?

Reply:
What if client decides to jump between forks? Do we just delete the history in this regard? 

I'm fine with that tbh. Let's assume sequentiality unless something else comes up. I dont want to over engineer this problem